Dockerfile

```diff
@@ -148,8 +148,7 @@ RUN luarocks install luacheck
 # NPM setup
 # Extract dependencies from coala-bear package.json
 # typescript is a peer dependency
-RUN npm install -g typescript \
-    $(sed -ne '/~/{s/^[^"]*"//;s/".*"~/@/;s/",*//;p}' coala-bears/package.json)
+RUN npm install
```

Comment:
uhm, I don't see you setting the workdir to `/coala-bears` or cding in there. This certainly doesn't seem to fix the markdown bear issue: https://travis-ci.org/coala/docker-coala-base/builds/188154908#L8595

And indeed: https://travis-ci.org/coala/docker-coala-base/builds/188154908#L2756